R/code_agreements.R

```diff
@@ -116,49 +133,59 @@ code_agreements <- function(dataset = NULL, title, date) {
 #' code_parties(IEADB$Title)
 #' @export
 code_parties <- function(title) {
-  
```

Comment:
For the future: if code_term and code_grounds are in their own scripts, should code_parties? Should we have them described in the same or different help files?

Reply:
`code_parties()` is called by `code_agreements()` and this is why it is on that script, we can change if you would like but then would we have a different script for all the exported functions called by `code_agreements()` or only for `code_parties()`? Perhaps `code_term()` and `code_grounds()` should be in the same script? I will open an issue for this and we can discuss it in a future PR.